app/role_data.go

```diff
@@ -48,6 +48,7 @@ func addRoles(management *config.ManagementContext) (string, error) {
 		addRule().apiGroups("management.cattle.io").resources("nodedrivers").verbs("get", "list", "watch").
 		addRule().apiGroups("management.cattle.io").resources("podsecuritypolicytemplates").verbs("get", "list", "watch").
 		addRule().apiGroups("management.cattle.io").resources("nodetemplates").verbs("*").
+		addRule().apiGroups("management.cattle.io").resources("multiclusterapps", "globaldnses").verbs("create").
```

Comment:
@mrajashree we should add the same rule for the globaldns

Reply:
Add globalDNSProvider